src/watertap_contrib/reflo/costing/units/air_stripping.py

```diff
@@ -0,0 +1,563 @@
+
+    make_capital_cost_var(blk)
+    blk.capital_cost.setlb(0)
+
+    ax = blk.unit_model
+    packing_material = ax.config.packing_material
+    prop_in = ax.process_flow.properties_in[0]
+    prop_out = ax.process_flow.properties_out[0]
+    ax_params = blk.costing_package.air_stripping
+    tower_height_ft = pyo.units.convert(ax.tower_height, to_units=pyo.units.feet)
+    tower_diam_in = pyo.units.convert(ax.tower_diam, to_units=pyo.units.inch)
+    flow_vol_air = pyo.units.convert(
+        pyo.units.convert(
+            prop_in.flow_vol_phase["Vap"],
+            to_units=pyo.units.m**3 / pyo.units.hr,
+        )
+        * pyo.units.hr
+        / pyo.units.m**3,
+        to_units=pyo.units.dimensionless,
+    )
+
+    base_currency = blk.config.flowsheet_costing_block.base_currency
+
+    # default packing material is PVC
+    if packing_material == "ceramic":
+        ax_params.capital_cost_packing.fix(2000)
+    if packing_material == "stainless_steel":
+        ax_params.capital_cost_packing.fix(8000)
+
+    blk.tower_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Aluminum tower cost",
+    )
+
+    blk.port_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Aluminum access port system cost",
+    )
+
+    blk.piping_liq_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Liquid inlet/outlet piping cost",
+    )
+
+    blk.piping_air_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Air inlet piping cost",
+    )
+
+    blk.tray_ring_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Tray ring cost",
+    )
+
+    blk.tray_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Tray cost",
+    )
+
+    blk.plate_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Plate cost",
+    )
+
+    blk.tower_internals_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Tower internals cost",
+    )
+
+    blk.packing_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Packing material cost",
+    )
+
+    blk.mist_eliminator_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Mist eliminator cost",
+    )
+
+    blk.pump_cost = pyo.Var(
+        initialize=1e5,
+        bounds=(0, None),
+        units=base_currency,
+        doc="Water pump cost",
+    )
+
+    blk.pump_power = pyo.Var(
+        initialize=50,
+        bounds=(0, None),
+        units=pyo.units.kilowatt,
+        doc="Water pump power requirement",
+    )
```

Comment:
I had a similar comment for blower_power; I think any Var that could hypothetically be used without cost (e.g., energy/power consumption, etc.) should live on the unit model rather than in it's costing module.